pkg/agent/api/delegatedidentity/v1/service.go

```diff
@@ -180,9 +180,14 @@ func sendX509SVIDResponse(update *cache.WorkloadUpdate, stream delegatedidentity
 	// a response has already been sent so nothing is
 	// blocked on this logic
 	for i, svid := range resp.X509Svids {
+		id, err := idutil.IDProtoString(svid.X509Svid.Id)
+		if err != nil {
+			log.WithError(err).Warn("Could not convert SPIFFEID proto to string")
+			continue
```

Comment:
Although it should never happen, we should still try to log the event in this case without the `spiffe_id` log field

Reply:
updated